plugin/PDFDetailsPage.qml

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2024 Damien Caliste
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Office 1.0
+
+DetailsPage {
+    DetailItem {
+        //: Page count of the PDF document
+        //% "Page Count"
+        label: qsTrId("sailfish-office-la-pdf-pagecount")
+        value: document.pageCount
+        alignment: Qt.AlignLeft
+        visible: !document.locked
+    }
+    SectionHeader {
+        visible: document.passwordProtected
+        //% "Read protection"
+        text: qsTrId("sailfish-office-la-pdf-readprotection")
+    }
+    Label {
+        visible: document.passwordProtected
+        width: parent.width - 2*x
+        x: Theme.horizontalPageMargin
+        //% "This document is protected by a password"
+        text: qsTrId("sailfish-office-la-pdf-passwordprotected")
+        color: palette.secondaryHighlightColor
+        font.pixelSize: Theme.fontSizeSmall
+        textFormat: Text.PlainText
+        wrapMode: Text.Wrap
+    }
+    Item {
+        visible: document.passwordProtected
+        width: parent.width
+        height: Theme.paddingMedium
```

Comment:
This still seemed a bit tight. Could you make it paddingLarge and also add one between the password field and the clear button.

Reply:
Sure, done (changing for padding large, and adding another padding large between the password field and the clear button).